benefits/eligibility/views.py

```diff
@@ -37,7 +29,7 @@ def index(request, agency=None):
         else:
             session.update(request, eligible=False, origin=agency.index_url)
     else:
-        session.update(request, agency=agency, eligible=False, origin=reverse(ROUTE_CORE_INDEX))
+        session.update(request, agency=agency, eligible=False, origin=agency.index_url)
```

Comment:
I think this is correct, since if the session has an agency, the origin is `agency.index_url` --> `routes.AGENCY_INDEX` --> `"core:agency_index"` (url `"/cst"`). But originally `origin` was set to `"core:index"` via `ROUTE_CORE_INDEX` (url `"/"`) so just wanted to make sure this change was intended.

Reply:
Yes, thanks for noticing this and asking. 

I did intentionally make this change because it seemed wrong to direct the user to `core:index` if they had an agency selected already.